acquire/acquire.py

```diff
@@ -1523,6 +1523,35 @@ class OSX(Module):
     ]
 
 
+@register_module("--osx-applications")
+class OSXApplications(Module):
+    DESC = "OS-X application files"
+    SPEC = [
+        # applications
+        ("glob", "/Applications/*/Contents/Info.plist"),
+        ("glob", "Applications/*/Contents/Info.plist", from_user_home),
+    ]
+
+    @classmethod
+    def get_spec_additions(cls, target):
+        spec = set()
+
+        for appplication_support in itertools.chain(
+            ["/Library/Application Support"],
+            from_user_home(target, "Library/Application Support"),
```

Comment:
Are there more specific paths you can use instead of every file <=1MB from these paths? Doing a quick `find` on my personal Mac results in around 30GB of data, much of which are cache files and Steam game files 😉.

Also, don't these paths generally contain a lot of privacy sensitive information (session cookies and the like)? Whereas the other application paths contain just generic installed application information. Perhaps nice to separate them, include the `Info.plist` info in every profile (minimal, default and full), and a little bit more fleshed out Application Support module in full?

Reply:
I have adjusted the pattern to only acquire files in directory and subdirectories of the applications, because these paths contain configuration files of the applications and are mostly used by attacker shows in the blog posts of Objective See.

The modules are seperated.

Reply:
With the used `glob("*/*/*")` you're now searching for files exactly two sub-directories deep (no more, no less). From your comment, I think that's not the behaviour you wish for, is it?

If you indeed want to search more recursively, you might want to use `rglob("*")` or `rglob("*/*")` instead. This allows you to search for all files within the "Application Support" directory and/or all it's sub-directories.

Reply:
@Poeloe I should have said the files in the directory of the application and files in the sub-directory of an application, as shown below.

* `/Users/<USER>/Library/Application Support/`
* `/Users/<USER>/Library/Application Support/app1`
* `/Users/<USER>/Library/Application Support/app1/dir/`

When using rglob the files in the following directory are also collected, which I don't want.

`/Users/<USER>/Library/Application Support/app1/dir/dir/dir/dir/dir`

Reply:
Maybe the solution is then to use `rglob` in combination with and if-statement like:

```python
if not path.is_file():
   continue

if path.parts[1] == "Library" and len(path.parts) > 5:
   continue

if len(path.parts) > 7:
   continue

if path.stat().st_size >= (1 * 1024 * 1024):  # 1MB
   continue
```

Does this give you the proper results?

Reply:
The current implementation [`25467c4` (#96)](https://github.com/fox-it/acquire/pull/96/commits/25467c40c766dc17b18e330466fe396a22b71191) already gives the proper results. Is it needed to use `rglob` with your suggested code?

Reply:
That's true, I have removed the module.

How would acquire ASDF solve the issue of not having the luxury of a MFT for Unix based systems?

Reply:
Once ASDF support is included, you'll be able to completely browse any filesystem for which we have added ASDF support. Iterating files and inodes, browsing around with `target-shell` and `target-query` plugins  like `suidbinaries` will all work as expected, while still maintaining the small collection footprint that acquire has now. It's like having a full disk image, but smaller.

In addition, files that you have collected (e.g. the `Info.plist`) will be completely readable. Any file that you will not have collected, on the other hand, will return `\xa5\xdf` as its file content.